common/store.js

```diff
@@ -1,4 +1,8 @@
-import { saveState, loadState, loadStatePropertyOrEmptyObject } from 'utils/localStorage';
+import {
+  saveState,
+  loadState,
+  loadStatePropertyOrEmptyObject
+} from 'utils/localStorage';
```

Comment:
@crptm Any ideas of we're still getting formatting mismatches if we're all using prettier?